_posts/2016/2016-08-29-sageone-approach.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "approach of sageone connector as a deep integrated app"
+date: "2016-08-29 07:00:28"
+image: blog-header/bridge.jpg
+categories: tech-stories
+authors: ["Mats"]
+---
+
+In the third-party team our recent and still ongoing project was the implementation of an application serving as a connector between the epages e-commerce system and Sage One. Sage One is an ERP solution. The main purpose of this connector-application is to transfer the orders being created in the epages shop into the sageone accounting application.
```

Comment:
In the third-party team our recent and still ongoing project was the implementation of an application serving as a connector between the epages e-commerce system and Sage One.

>
In my team the recent and still ongoing project is the implementation of an application serving as a connector between the ePages e-commerce solution and Sage One.